<launch>
  <!-- <include file="$(find pr2_3dnav)/both_arms_navigation.launch"/> -->
  
  <!-- <include file="$(find pr2_2dnav_slam)/launch/pr2_2dnav.launch/> -->
  <node pkg="pr2_tuck_arms_action" type="tuck_arms.py" name="tuck_arms_action" output="screen">
  </node>
  <node pkg="pyride_pr2" type="pyride_pr2" name="pyride_pr2" output="screen">
    <remap from="cmd_vel" to="base_controller/command" />
    <remap from="head_vel" to="head_traj_controller/command" />
    <param name="config_file" type="str" value="$(find pyride_pr2)/pyrideconfig.xml"/>
    <param name="script_dir" type="str" value="$(find pyride_pr2)/scripts"/>
    <param name="use_moveit" type="bool" value="false"/>
  </node>
</launch>
